### PR TITLE
Blog for HA

### PR DESCRIPTION
**Pull request by muralibasani**

### CODE REVIEW

File: docs/blog/2023/05/high-availability-for-klaw.rst [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+Klaw organizes data in the database into three categories:
+
+    Admin data: Comprises users, roles, permissions, teams, tenants, clusters, environments, properties and other product related details.
+
+    Core data: Comprises Topics, ACLs, Schemas and Connector configurations.
+
+    Requests data: Comprises requests of Topics, ACLs, Schemas and Connectors.
+
+- Cache
+Klaw stores most authorization-related data in the cache to avoid humongous database calls. This effectively reduces latency and gives users immediate response from the application. However, this cache is reset whenever changes are requested.
```
Comment (aindriu-aiven) [inline docs/blog/2023/05/high-availability-for-klaw.rst:112]: Maybe 
"Klaw stores most frequently queried data in a local cache to for improved performance and user experience." ?

File: docs/blog/2023/07/high-availability-for-klaw.rst [hunk 2]
```diff
@@ -0,0 +1,270 @@
+
+High availability for Klaw
+==========================
+
```
Comment (aindriu-aiven) [inline docs/blog/2023/07/high-availability-for-klaw.rst:4]: Just noticed this is repeated twice one in the Mya folder and one in july.

File: docs/blog/2023/07/high-availability-for-klaw.rst [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+.. post:: July 4, 2023
+   :tags: kafka,governance,topics,acls,klaw, high availability, fault tolerance
+   :category: klaw
+   :author: Murali Basani
+
+Background
+----------
+In today's dynamic and ever-changing digital landscape, maintaining the uninterrupted operation of applications is of the highest importance. This article explores how to configure Klaw for high availability in production environments, enhancing reliability and delivering a seamless user experience.
+
+In this blog post, we will dive deep into Klaw—an Java-based application explicitly created for Kafka governance. We will focus on exploring the configuration process for achieving high availability (HA) in production environments. By implementing these measures, we can ensure continuous operation, bolster reliability, and deliver a seamless user experience, even in unexpected failures.
```
Comment (harshini-rangaswamy) [inline docs/blog/2023/07/high-availability-for-klaw.rst:14]:
```suggestion
In today's dynamic and ever-changing digital landscape, maintaining the uninterrupted operation of applications is of the highest importance. This blog post explores the configuration of Klaw, a Java-based application designed specifically for Kafka governance. Our primary focus is on exploring the configuration process for achieving high availability (HA) in production environments. By implementing these measures, we can ensure continuous operation, enhance reliability, and deliver a seamless user experience, even during unexpected failures.
```

Comment (harshini-rangaswamy) [inline docs/blog/2023/07/high-availability-for-klaw.rst:14]: Shortened this section to improve clarity. 
```suggestion
In today's dynamic and ever-changing digital landscape, maintaining the uninterrupted operation of applications is of the highest importance. This blog post explores the configuration of Klaw, a Java-based application designed specifically for Kafka governance. 

Our primary focus is on exploring the configuration process for achieving high availability (HA) in production environments. By implementing these measures, we can ensure continuous operation, enhance reliability, and deliver a seamless user experience, even during unexpected failures.
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]

File: docs/blog/2023/07/high-availability-for-klaw.rst (hunk 3 above)
Comment (harshini-rangaswamy) [reply docs/blog/2023/07/high-availability-for-klaw.rst:14]: @muralibasani update the content for the "Background" section. We used a mix of words like article and blogs, just wanted to keep the words consistent.

Comment (aindriu-aiven) [inline docs/blog/2023/07/high-availability-for-klaw.rst:14]: Sorry missed this the first time
"Klaw—an Java-based" -> Klaw—a Java-based

**Review by harshini-rangaswamy — APPROVED** [review]

File: docs/blog/2023/07/high-availability-for-klaw.rst [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+
+
+Klaw deployment model in high availability (HA) mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In the high availability mode, Nginx routes the requests to Klaw instances which are connected to a common data store such as Postgres. Each Klaw instance comprises both the Governance and Cluster API applications. You may choose to deploy these applications on separate machines for increased robustness.
+Below is the deployment model of Klaw in HA mode.
+
+
+Klaw in HA mode with Nginx Load balancer
```
Comment (harshini-rangaswamy) [inline docs/blog/2023/07/high-availability-for-klaw.rst:242]: @muralibasani Just realized that there is an image missing here. Apart from this, the rest looks good.

Comment (muralibasani) [reply docs/blog/2023/07/high-availability-for-klaw.rst]: @harshini-rangaswamy thank you. Added the image now.

**Pull request merged**